sw/host/tests/manuf/manuf_cp_test_lock/src/main.rs

```diff
@@ -0,0 +1,171 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! This test performs a lifecycle transition from `TEST_UNLOCKED0` to `TEST_LOCKED0`.
+//!
+//! We also test provisioning a `TEST_UNLOCKED` being programmed into OTP.
+
+use std::rc::Rc;
+
+use anyhow::Context;
+use structopt::StructOpt;
+
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::dif::lc_ctrl::{DifLcCtrlState, LcCtrlReg};
+use opentitanlib::dif::otp_ctrl::{DaiParam, Partition};
+use opentitanlib::execute_test;
+use opentitanlib::io::jtag::{Jtag, JtagParams, JtagTap};
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::test_utils::lc_transition;
+use opentitanlib::test_utils::otp_ctrl::{OtpParam, OtpPartition};
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    #[structopt(flatten)]
+    init: InitializeTest,
+    #[structopt(flatten)]
+    jtag: JtagParams,
+}
+
+/// Pre-image of the TEST_UNLOCK token that will be written to OTP.
+const TEST_UNLOCK_TOKEN_PREIMAGE: [u32; 4] = [0xAAAAAAAA, 0xAAAAAAAA, 0xAAAAAAAA, 0xAAAAAAAA];
+/// Post-image of the TEST_UNLOCK token as generated by `gen_test_unlock_token.py`.
+const TEST_UNLOCK_TOKEN_POSTIMAGE: [u32; 4] = [0xe6b9ec3f, 0x67fd75f3, 0x618ae065, 0xc0bb8f81];
+
+fn main() -> anyhow::Result<()> {
```

Comment:
nit: import "anyhow::Result"?

Reply:
Sure. I only write these out to avoid confusion with the standard `Result` from the prelude, but don't mind importing it.